infinite-tracing/src/main/java/com/newrelic/SpanEventConsumer.java

```diff
@@ -45,7 +48,9 @@ private SpanEventConsumer(BlockingQueue<SpanEvent> queue, MetricAggregator aggre
     @Override
     public void accept(SpanEvent spanEvent) {
         aggregator.incrementCounter("Supportability/InfiniteTracing/Span/Seen");
-        queue.offer(spanEvent);
+            if(!queue.offer(spanEvent)) {
```

Comment:
Currently, we create a metric for `Seen`  vs `Sent` .  The difference between the two are the number of spans that are presumably dropped. 

I think it would be nice to have a log message to indicate if the queue is full